config/ember-try.js

```diff
@@ -6,18 +6,18 @@ module.exports = async function() {
   return {
     scenarios: [
       {
-        name: 'ember-lts-3.8',
+        name: 'ember-lts-3.12',
```

Comment:
I still want to support Ember 3.8. That's a huge breaking change :/

Reply:
I can remove `@glimmer/tracking` and revert the ember-try config

This was all automatically done when running `ember-cli-update --to 3.16.2`

There's no need to drop support if there are no breaking changes 👀